### PR TITLE
[minor] Add role to renew MAS certificates

### PR DESCRIPTION
**Pull request by JonahLuckett**
For issue: #327

Initial work on creating a new role that will renew a certificate and check that the new private key created is different from the old one - assertions will fail currently as this is preliminary work to prepare MAS keys for new rotationPolicy settings and the updated private key sizings

See original issue at: https://github.ibm.com/wiotp/tracker/issues/9081

### CODE REVIEW

File: ibm/mas_devops/roles/cert_renew_check/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Role Variables
+--------------
+### mas_instance_id
+Required - Defines the instance id that is used in the existing MAS installation, will be used to lookup the existing MAS subscription and targeted Certificates, Issuers and Secrets to be migrated.
+
+- Environment Variable: `MAS_INSTANCE_ID`
+- Default Value: None
+
+### custom_cluster_issuer
+Optional - Only required if the targeted MAS 8.6 instance is configured with a custom cluster issuer (i.e Let's Encrypt).
```
Comment (durera) [inline ibm/mas_devops/roles/cert_renew_check/README.md:15]: Is there something specific to MAS 8.6 here?

**Review by durera — CHANGES_REQUESTED** [review]
- Can we also refactor the name to `suite_certs_renew`
- Need to add an entry to build/bin/copy-role-docs
- Need to add an entry to mkdocs.yml

Looks solid otherwise :+1:

File: ibm/mas_devops/roles/cert_renew_check/README.md (hunk 1 above)
Comment (JonahLuckett) [reply ibm/mas_devops/roles/cert_renew_check/README.md]: Ah no - in fact I didn't use the custom cluster issuer so have removed it from the readMe

**Comment by JonahLuckett** [comment]
Have added requested changes

**Review by durera — CHANGES_REQUESTED** [review]
See comments in Slack :+1:

**Review by durera — CHANGES_REQUESTED** [review]
Tested the new role, and it fails with the below error ...

`ROLE_NAME=suite_certs_renew ansible-playbook ibm.mas_devops.run_role`

```
...
TASK [ibm.mas_devops.suite_certs_renew : Lookup original fvtstable-admindashboard-cert-internal data] ************************************************************************************
Wednesday 09 November 2022  10:27:19 +0000 (0:00:00.101)       0:00:05.363 ****
ok: [localhost]

TASK [ibm.mas_devops.suite_certs_renew : Run cmctl renew command] ************************************************************************************************************************
Wednesday 09 November 2022  10:27:21 +0000 (0:00:01.125)       0:00:06.489 ****
changed: [localhost]

TASK [ibm.mas_devops.suite_certs_renew : Wait for fvtstable-admindashboard-cert-internal to be renewed (30s delay)] **********************************************************************
Wednesday 09 November 2022  10:27:21 +0000 (0:00:00.750)       0:00:07.239 ****
ok: [localhost]

TASK [ibm.mas_devops.suite_certs_renew : Assert new 'tls.crt' in fvtstable-admindashboard-cert-internal secret is different from the one prior to renewal] *******************************
Wednesday 09 November 2022  10:27:22 +0000 (0:00:01.163)       0:00:08.403 ****
fatal: [localhost]: FAILED! => {"msg": "The conditional check 'internalcert_original_data.resources[0].data['tls.crt'] != new_secret_result.resources[0].data['tls.crt']' failed. The error was: error while evaluating conditional (internalcert_original_data.resources[0].data['tls.crt'] != new_secret_result.resources[0].data['tls.crt']): 'new_secret_result' is undefined"}

NO MORE HOSTS LEFT ***********************************************************************************************************************************************************************

PLAY RECAP *******************************************************************************************************************************************************************************
localhost                  : ok=31   changed=2    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0
```

**Pull request closed without merging**